device-types/Dell/PowerSwitch-N2248X-ON.yaml

```diff
@@ -0,0 +1,131 @@
+---
+manufacturer: Dell
+model: PowerSwitch N2248X-ON
+slug: dell-powerswitch-n2248x-on
+part_number: N2248X-ON
+u_height: 1
+is_full_depth: true
+console-ports:
+  - name: Console
+    label: RJ-45 Console
+    type: rj-45
+  - name: USB
+    label: USB Console
+    type: usb-micro-b
+power-ports:
+  - name: PSU1
+    type: iec-60320-c14
+  - name: PSU2
+    type: iec-60320-c14
```

Comment:
It appears that these PSUs are modular, is this true?

If so, you should remove these definitions and create `module-bays` for them to slot into. Then, if the modules do not already exist in this repo you should add them to the associated module-types folder